Javascript/Ez Namey /namey.js

```diff
@@ -0,0 +1,43 @@
+let firstWords = [
+  "Amazing",
+  "Creative",
+  "Super",
+  "Fantastic",
+  "Smart",
+  "Future",
+  "Ultimate",
+  "Generation",
+  "Brilliant",
+  "Innovative",
+];
+let secondWords = [
+  "Solutions",
+  "Ideas",
+  "Concepts",
+  "Technologies",
+  "Systems",
+  "Creations",
+  "Inventions",
+  "Platforms",
+  "Ventures",
+  "Projects",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
+let nameLength = startupName.length;
```

Comment:
Use `const` until it doesn't `let` you to ;)

Reply:
***Changed.***